packages/@lwc/compiler/README.md

```diff
@@ -125,6 +125,18 @@ export interface CompilerOptions {
     outputConfig?: OutputConfig;
 }
 
+export interface TransformOptions {
+    name: string;
+    namespace: string;
+    /**
+     * An optional directory prefix that contains the specified components
+     * files. Only used when the component that is the compiler's entry point.
```

Comment:
do we need 'that' in this last sentence? > Only used when the component is the compiler's entry point